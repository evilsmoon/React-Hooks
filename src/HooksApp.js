import React from 'react'

export const HooksApp = () => {
    return (
        <div>

            <h1>Hello word</h1>

        </div>
    )
}
