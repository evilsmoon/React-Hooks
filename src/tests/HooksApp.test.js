import '@testing-library/jest-dom'
import React from 'react';
import { shallow } from 'enzyme'
import { HooksApp } from '../HooksApp'

describe('Test HooksApp.js', () => {

    test('mostrar <HooksApp/>', () => {

        const wrapper = shallow(<HooksApp/>);
 
        expect(wrapper).toMatchSnapshot();
        
    })
    
    
})
