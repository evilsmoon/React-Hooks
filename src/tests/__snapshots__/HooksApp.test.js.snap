// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HooksApp.js mostrar <HooksApp/> 1`] = `
<div>
  <h1>
    Hello word
  </h1>
</div>
`;
